### PR TITLE
Options for handling multi-channel RIR

### PR DESCRIPTION
**Pull request by desh2608**
`rir_channels` option can be specified with `reverb_rir` or `ReverbWithImpulseResponse` to select which channels to use in case of multi-channel RIR. If base cut is MonoCut and RIR is multi-channel, it produces a MixedCut with number of tracks equal to the number of RIR channels.

### CODE REVIEW

File: lhotse/cut.py [hunk 1]
```diff
@@ -1976,6 +2015,7 @@ def reverb_rir(
         normalize_output: bool = True,
         early_only: bool = False,
         affix_id: bool = True,
+        keep_mono: bool = False,
```
Comment (pzelasko) [inline lhotse/cut.py:2018]: Can we make it `True` by default? For simple augmentation in scenarios like ASR / pre-training, it seems like the right thing to do.

**Comment by pzelasko** [comment]
Looks good to me! Can I merge?

**Comment by desh2608** [comment]
> Looks good to me! Can I merge?

Yeah, it's ready to merge :)

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**